.github/workflows/build.yml

```diff
@@ -34,6 +33,18 @@ jobs:
           else
             git checkout -b releases origin/releases
           fi
+
+      - name: Debug changelog file
+        run: |
+          ls -la ${{ github.workspace }}/CHANGELOG.txt
+          cat ${{ github.workspace }}/CHANGELOG.txt
+  
+      - name: Collect commit ranges
```

Comment:
Moves commit log collection before we actually change the contents of the checkout of releases.